go/oasis-test-runner/cmd/root.go

```diff
@@ -422,6 +422,9 @@ func doScenario(childEnv *env.Env, sc scenario.Scenario) (err error) {
 		return
 	}
 
+	// Enable shorter per-node socket paths.
+	fixture.Network.UseCustomGrpcSocketPath = true
```

Comment:
This will panic as fixture can be `nil` in cases where `sc.Init` sets up a custom network. Additionally it makes less sense to have this in the fixture as you probably want to always enable this when running in the test runner (as otherwise the tests may just fail).

Reply:
Understood. But I also can't use `net.cfg` as it can also be `nil` or not?

Reply:
I tested and `net.cfg` is always defined.